Documentation/Settings.cfg

```diff
@@ -0,0 +1,53 @@
+# Sphinx setup file for this project
+
+[general]
+
+project     = phpMyAdmin for TYPO3
+version     = 5.3
+release     = 5.3.0
```

Comment:
Shouldn't this be
version = 5.2
release = 5.2.7
?

Reply:
Ah, okay, next merged commit increases major version. :-)

Reply:
Yepp, I did'nt want to mix the security release with the documentation change so I split the commit but missed that one. I strongly suggest to use the 5.3 release ;-)